src/simulation/thermal/sensorThermal/sensorThermal.cpp

```diff
@@ -110,7 +106,7 @@ bool SensorThermal::readMessages()
         this->sensorStatus = 1;
     }
 
-    return(true);
+    return;
```

Comment:
as we are not actually returning anything, please delete this `return` statement

Reply:
Done.